src/features/allusers/allusers.controller.js

```diff
@@ -0,0 +1,20 @@
+export default function AllUsersController($rootScope, $scope, $http) {
+ 
+    let user = $rootScope.globals.currentUser;
+
+    $scope.username = user.username;
+    $scope.currentNavItem = "allUsers";
```

Comment:
Powinno być:
`$scope.currentNavItem = "users";`

Gdy masz
```html
<md-nav-item md-nav-href="/allUsers" name="users">
   All users
</md-nav-item>
```
to `currentNavItem` powinno być równe nazwie, którą masz w `name`. Wtedy jak wejdziesz na podstronę `/allUsers` to zakladka "All users" będzie podświetlona :)